src/panel/category_panel.js

```diff
@@ -111,10 +113,9 @@ export default class CategoryPanel {
   }
 
   selectPoi(poi){
-    // TODO telemetry
     fire('fit_map', poi, layouts.LIST)
     this.close(false)
-    PanelManager.loadPoiById(poi.id, {isFromList : true, list: this})
+    PanelManager.loadPoiById(poi.id, {isFromList : true, list: this, source: 'pagesjaunes'})
```

Comment:
Wouldn't it be better if we create a constant object for the poi sources ?

Reply:
Indeed, done